doc/sources/quick-start.rst

```diff
@@ -14,17 +14,455 @@
+
+Install from PyPI 
+**********************
+
+Recommended by default. 
+
+To install |intelex|, run:
+
+::
+
+  pip install scikit-learn-intelex
+
+**Supported Configurations**
+
+.. list-table::
+   :header-rows: 1
+   :align: left
+
+   * - OS / Python version
+     - Python 3.8
+     - Python 3.9
+     - Python 3.10
+     - Python 3.11
+     - Python 3.12
+   * - Linux* OS
+     - [CPU, GPU]
+     - [CPU, GPU]
+     - [CPU, GPU]
+     - [CPU, GPU]
+     - [CPU, GPU]
+   * - Windows* OS
+     - [CPU, GPU]
+     - [CPU, GPU]
+     - [CPU, GPU]
+     - [CPU, GPU]
+     - [CPU, GPU]
+
+
+
+Install from Anaconda* Cloud
+********************************************
+
+To prevent version conflicts, we recommend installing `scikit-learn-intelex` into a new conda environment.
+
+.. tabs::
+
+   .. tab:: Conda-Forge channel
+
+      Recommended by default. 
+      
+      To install, run::
+
+        conda install scikit-learn-intelex -c conda-forge
+      
+      .. list-table:: **Supported Configurations**
+         :header-rows: 1
+         :align: left
+
+         * - OS / Python version
+           - Python 3.8
+           - Python 3.9
+           - Python 3.10
+           - Python 3.11
+           - Python 3.12
+         * - Linux* OS
+           - [CPU]
+           - [CPU]
+           - [CPU]
+           - [CPU]
+           - [CPU]
+         * - Windows* OS
+           - [CPU]
+           - [CPU]
+           - [CPU]
+           - [CPU]
+           - [CPU]
+
+
+   .. tab:: Intel channel
+
+      Recommended for the Intel® Distribution for Python users. 
+
+      To install, run::
+
+        conda install scikit-learn-intelex -c intel
+      
+      .. list-table:: **Supported Configurations**
+         :header-rows: 1
+         :align: left
+
+         * - OS / Python version
+           - Python 3.8
+           - Python 3.9
+           - Python 3.10
+           - Python 3.11
+           - Python 3.12
+         * - Linux* OS
+           - [CPU, GPU]
+           - [CPU, GPU]
+           - [CPU, GPU]
+           - [CPU, GPU]
+           - [CPU, GPU]
+         * - Windows* OS
+           - [CPU, GPU]
+           - [CPU, GPU]
+           - [CPU, GPU]
+           - [CPU, GPU]
+           - [CPU, GPU]
+ 
+
+
+   .. tab:: Main channel
+
+      To install, run::
+
+        conda install scikit-learn-intelex
+      
+      .. list-table:: **Supported Configurations**
+         :header-rows: 1
+         :align: left
+
+         * - OS / Python version
+           - Python 3.8
+           - Python 3.9
+           - Python 3.10
+           - Python 3.11
+           - Python 3.12
+         * - Linux* OS
+           - [CPU]
+           - [CPU]
+           - [CPU]
+           - [CPU]
+           - [CPU]
+         * - Windows* OS
+           - [CPU]
+           - [CPU]
+           - [CPU]
+           - [CPU]
+           - [CPU]
+
+
+
+Build from Sources
```

Comment:
this is for sure not a quick start - we probably should move it to more dev oriented section and have a mention here that you can build from sources and here is link

Reply:
Maybe then keep the INSTALL.md file and just provide a link to it?

Reply:
yes, this would work